sdk/nodejs/runtime/closure/codePaths.ts

```diff
@@ -310,10 +313,10 @@ function findDependency(root: readPackageTree.Node | undefined | null, name: str
             // from the `path` property instead. Match any name that ends with something that looks
             // like `@foo/bar`, such as `node_modules/@foo/bar` or
             // `node_modules/baz/node_modules/@foo/bar.
-            const childFolderName = filepath.basename(child.path);
-            const parentFolderName = filepath.basename(filepath.dirname(child.path));
+            const childFolderName = upath.basename(child.path);
+            const parentFolderName = upath.basename(upath.dirname(child.path));
             if (parentFolderName[0] === "@") {
-                childName = `${parentFolderName}/${childFolderName}`;
+                childName = upath.joinSafe(parentFolderName, childFolderName);
```

Comment:
actually... joinSafe is not needed here... since we know parentFolderName starts with  '@'.  so i'll just remove.